lib/components/result-view/result-view.js

```diff
@@ -133,31 +133,38 @@ class ResultViewComponent extends React.Component<Props> {
             : { maxWidth: `${position.editorWidth}px`, margin: "0px" }
         }
       >
-        <Display
+        <div
+          className="hydrogen_cell_display"
           ref={ref => {
-            if (!ref || !ref.el) return;
-            this.el = ref.el;
+            if (!ref) return;
+            this.el = ref;
 
             isPlain
-              ? this.addCopyTooltip(ref.el, this.containerTooltip)
+              ? this.addCopyTooltip(ref, this.containerTooltip)
               : this.containerTooltip.dispose();
 
-            // React's event handler doesn't properly handle event.stopPropagation() for
-            // events outside the React context. Using proxy.el saves us a extra div.
-            // We only need this in the text editor, therefore we check showStatus.
-            if (!this.expanded.get() && !isPlain && ref.el) {
-              ref.el.addEventListener("wheel", this.onWheel(ref.el), {
+            // As of this writing React's event handler doesn't properly handle
+            // event.stopPropagation() for events outside the React context.
+            if (!this.expanded.get() && !isPlain && ref) {
+              ref.addEventListener("wheel", this.onWheel(ref), {
                 passive: true
               });
             }
           }}
-          outputs={toJS(outputs)}
-          displayOrder={displayOrder}
-          transforms={transforms}
-          theme="light"
-          models={{}}
-          expanded={this.expanded.get()}
-        />
+          style={{
+            maxHeight: this.expanded.get() ? "100%" : `${SCROLL_HEIGHT}px`,
+            overflowY: "auto"
+          }}
+        >
+          <Display
+            outputs={toJS(outputs)}
+            displayOrder={displayOrder}
+            transforms={transforms}
+            theme="light"
+            models={{}}
```

Comment:
With the latest version of the display area you no longer have to declare models or theme.